Cargo.toml

```diff
@@ -1,9 +1,10 @@
 [package]
 name = "artichoke"
-version = "0.1.0-pre.0"
+version = "0.2.0-pre.0"
 authors = ["Ryan Lopopolo <rjl@hyperbo.la>"]
 license = "MIT"
 edition = "2021"
+rust-version = "1.56.0"
```

Comment:
I like the idea of adding this bit of metadata, but I'd love to do it to all crates in this workspace and maybe automate bumping these fields along with the version in `rust-toolchain` every time the Rust compiler is released. Maybe with a Ruby script or a Rake task?

Let's explore this in a new PR.

Reply:
Filed an issue for this here: 

- https://github.com/artichoke/artichoke/issues/1581